acceptance/tests/vault/vault_test.go

```diff
@@ -65,7 +66,7 @@ func TestVault(t *testing.T) {
 		"global.secretsBackend.vault.connectCA.rootPKIPath":         "connect_root",
 		"global.secretsBackend.vault.connectCA.intermediatePKIPath": "dc1/connect_inter",
 
-		"global.acls.manageSystemACLs":       "true",
+		"global.acls.manageSystemACLs":       "false",
```

Comment:
@ishustava When I talked to Ashwin the other day he was telling me that the ACLs were not needed because it doesn't change the TLS behaviour.

I did spend the time to look into ACLs and how to do it for my learning. So, if you want, I could still do it with ACLs and use the bootstrap token.. etc.